app/frontend/src/components/forms/manage/ManageFormActions.vue

```diff
@@ -93,13 +93,17 @@ export default {
     canViewSubmissions() {
       const perms = [
         FormPermissions.SUBMISSION_READ,
-        FormPermissions.SUBMISSION_UPDATE
+        FormPermissions.SUBMISSION_UPDATE,
```

Comment:
is this comma required?

Reply:
No it's not, and personally prefer it to not be there, but our VSCode autoformatter seems to insist these trailing end commas be there.